doc/report.xsd

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema targetNamespace="https://phpcq.github.io/v1/report.xsd"
+           xmlns:tns="https://phpcq.github.io/v1/report.xsd"
+           elementFormDefault="qualified"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema">
+
+  <xs:element name="report" type="tns:reportFile">
+    <xs:annotation>
+      <xs:documentation>
+        The root element for reports.
+      </xs:documentation>
+    </xs:annotation>
+  </xs:element>
+
+  <xs:complexType name="reportFile">
+    <xs:sequence>
+      <xs:element name="tools" type="tns:toolListType">
+      </xs:element>
+    </xs:sequence>
+    <xs:attribute name="status" type="tns:reportStatusType" />
+    <xs:attribute name="started_at" type="xs:dateTime"/>
+    <xs:attribute name="completed_at" type="xs:dateTime"/>
+    <xs:attribute name="checkstyleFile" type="xs:string"/>
```

Comment:
I'd drop the checkstyle information here as the generated reports should depend on the command configuration